src/ngraph_builder.cc

```diff
@@ -2513,6 +2514,48 @@ static Status TranslateQuantizeAndDequantizeV2Op(
   return Status::OK();
 }
 
+static Status TranslateQuantizedMaxPoolOp(
+    const Node* op, const std::vector<const Tensor*>& static_input_map,
+    Builder::OpMap& ng_op_map) {
+  shared_ptr<ng::Node> ng_input, ng_min, ng_max;
+  TF_RETURN_IF_ERROR(GetInputNodes(ng_op_map, op, &ng_input, &ng_min, &ng_max));
+  std::vector<int32> tf_strides;
+  std::vector<int32> tf_ksize;
+  std::string tf_padding_type;
+  TF_RETURN_IF_ERROR(GetNodeAttr(op->attrs(), "strides", &tf_strides));
+  TF_RETURN_IF_ERROR(GetNodeAttr(op->attrs(), "ksize", &tf_ksize));
+  TF_RETURN_IF_ERROR(GetNodeAttr(op->attrs(), "padding", &tf_padding_type));
+  bool is_nhwc = true;  // TODO, is this correct?
```

Comment:
Just curious, what does the TODO mean here?

Reply:
So the documentation says:
```
input: The 4D (batch x rows x cols x depth) Tensor to MaxReduce over.
```
which means it seems to expect input only in NHWC.

This is unlike normal Maxpool, which can accept a data format.

Kept a TODO as a note that QMP differs from MP in that respect, and a note that we assume NHWC, but might need to revisit in the future